### PR TITLE
Push up to date

### PR DESCRIPTION
**Pull request by jgrund**
Add all commits up to master

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "presets": [
+    [
+      "env",
+      {
+        "targets": {
+          "node": "current"
+        }
```
Comment (johnsonw) [inline .babelrc]: So this probably doesn't matter for this repo but in our other repos we've been putting this in as well:
```
 "browsers": [
            "last 1 chrome version",
            "last 1 firefox version"
          ]
```

File: .eslintignore [hunk 2]
```diff
@@ -1,2 +1,4 @@
-coverage
+dist
 node_modules
+coverage
+bin/test.js
```
Comment (johnsonw) [inline .eslintignore]: Do we need bin/test.js?

File: .eslintignore [hunk 3]
```diff
@@ -1,2 +1,4 @@
-coverage
+dist
```
Comment (johnsonw) [inline .eslintignore]: Do we need "dist"?

File: source/buffer-request.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
+// @flow
+
+//
+// INTEL CONFIDENTIAL
+//
+// Copyright 2013-2016 Intel Corporation All Rights Reserved.
```
Comment (johnsonw) [inline source/buffer-request.js]: We should probably update the copyright year in these files while we are in here.

File: .babelrc (hunk 1 above)
Comment (jgrund) [inline .babelrc]: We won't be using this in a browser, so we don't need that field.

File: .eslintignore (hunk 2 above)
Comment (jgrund) [inline .eslintignore]: No, removing.

File: .eslintignore (hunk 3 above)
Comment (jgrund) [inline .eslintignore]: No, removing.

File: source/buffer-request.js (hunk 4 above)
Comment (jgrund) [inline source/buffer-request.js]: Doesn't matter yet, the whole header will be changing shortly.

**Pull request merged**